MetaMask/metamask-mobile
# test: ramps order min and max limits

## **Description**

This adds E2E tests for the min and max limits when placing Buy orders 

## **Related issues**

https://consensyssoftware.atlassian.net/issues/RAMPS-1928

## **Manual testing steps**

All Ramps tests are passing:
```

 PASS  e2e/specs/ramps/onramp.spec.js (84.882 s)
  smokeAssets Buy Crypto
    ✓ should select Region and Payment Method to see the Build Quote screen (17662 ms)
    ✓ should enter the min and max order limits and verify the error messages (43375 ms)

Test Suites: 1 passed, 1 total
Tests:       2 passed, 2 total

```

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
